recipe/meta.yaml

```diff
@@ -31,9 +31,7 @@ requirements:
     # optional deps [socks]
     - pysocks >=1.5.6,<2.0,!=1.5.7
     # optional deps [brotli]
-    # issue with brotli and brotlicffi, so revert to brotlipy
-    # https://github.com/AnacondaRecipes/urllib3-feedstock/issues/2
-    - brotlipy >=0.6.0
+    - brotli-python >=1.0.9
```

Comment:
Does this cover `brotlicffi` and `brotli`?

Reply:
bortli-python is how we name pypa brotli in conda. (Because we also provide the library brotli.)
brotlicffi is not for cpython.